tests/tests/test-precompile/test-precompile-wormhole.ts

```diff
@@ -257,10 +257,6 @@ describeDevMoonbeam(`Test local Wormhole`, (context) => {
     console.log("Versioned User Action SCALE:", versionedUserAction.toHex());
     let payload = "" + versionedUserAction.toHex();
 
-    // TODO: very ugly hack
-    // replace byte 5 with 0x03. This seems to replace "v2" above with 3. Not sure what the issue is
-    payload = payload.substring(0, 5) + "3" + payload.substring(6);
-
```

Comment:
This was fixed (fc50a06) by using V1. I think the problem is that this index scheme isn't properly handled: https://github.com/PureStake/polkadot/blob/41fe2d1785f75150e57ca5dbf85e097f39f29401/xcm/src/lib.rs#L251